taxi_service/settings.py

```diff
@@ -140,3 +142,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
Make sure you've added a blank line at the end of the file to comply with the code style guidelines.